### PR TITLE
README.md accessibility touchups: language and link alts

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Description

* Don't use words like "just" and "simple" in docs; it's unintentionally mean to folks who wouldn't find things just simple
* Links should have descriptive alt text so that screenreaders can narrate them well as a user <kbd>Tab</kbd>s through the page

### Type of Change

- [ ] Adding a new entry
- [ ] Editing an existing entry (fixing a typo, bug, issues, etc)
- [x] Updating the documentation

### Checklist

- [x] My entry follows the Codecademy Docs style guide
- [x] I have performed a self-review of my own writing and code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my entry and corrected any misspellings
- [x] All writings are my own

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,18 +25,19 @@ Before contributing, please read through the [Contribution Guide](https://github
 
 ## How do I contribute?
 
-Contributing is as simple as:
+Thanks for your interest; we'd love to work with you! 💖
```
Comment (sonnynomnom) [inline README.md:28]: Love it.

File: README.md [hunk 2]
```diff
@@ -25,18 +25,19 @@ Before contributing, please read through the [Contribution Guide](https://github
...
 4. Making a pull request to merge your fork with this repo.
 
-If you haven't gone through this workflow before, you can check out [this YouTube video](https://www.youtube.com/watch?v=rgbCcBNZcdQ) to learn about how to make a pull request from a fork using Git. Alternatively, if you'd prefer to keep things to the GitHub UI for now, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a pull request by pressing this handy button:<br>
+If you haven't gone through this workflow before, you can check out <a alt="YouTube: Creating a Simple Github Pull Request" href="https://www.youtube.com/watch?v=rgbCcBNZcdQ">this YouTube video</a> to learn about how to make a pull request from a fork using Git. Alternatively, if you'd prefer to keep things to the GitHub UI for now, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a pull request by pressing this handy button:<br>
```
Comment (sonnynomnom) [inline README.md:36]: Wait, is there a difference between the `alt` attribute using the `` element and a title link using Markdown?

Link Title: https://www.codecademy.com/resources/docs/markdown/links

**Review by sonnynomnom — COMMENTED** [review]
Left one note!

File: README.md (hunk 2 above)
Comment (sonnynomnom) [reply README.md:36]: @JoshuaKGoldberg

Comment (JoshuaKGoldberg) [reply README.md:36]: The difference here is that both links had the same, relatively uninformative link title. Someone receiving a list of all links in the readme would get something like:

* "Codecademy Docs"
* "Contribution Guide"
* "this YouTube video"
* "this YouTube video"

The first two are clear, but the last two aren't. https://www.w3.org/TR/WCAG20-TECHS/H33.html

Ironically my explanation wasn't very clear: thanks for asking! 😄 

> a difference between the `alt` attribute ...

D'oh, I wasn't thinking; I should be using `title`! (per the w3 doc)

Comment (sonnynomnom) [reply README.md]: Dope! I think we can also do this:

`[link text](URL "link title")`

Anyway, merging!

File: README.md [hunk 3]
```diff
@@ -25,18 +25,19 @@ Before contributing, please read through the [Contribution Guide](https://github
...
+If you haven't gone through this workflow before, you can check out <a href="https://www.youtube.com/watch?v=rgbCcBNZcdQ" title="YouTube: Creating a Simple Github Pull Request">this YouTube video</a> to learn about how to make a pull request from a fork using Git. Alternatively, if you'd prefer to keep things to the GitHub UI for now, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a pull request by pressing this handy button:<br>
 
 <img src="https://github.com/Codecademy/docs/blob/main/media/pull-request-ui.png" alt="Code block with Codebyte tags" width="800"/>
 
-If you are uncomfortable using Git, you can also check out [this YouTube video](https://youtu.be/RPagOAUx2SQ) to do this all using the GitHub Desktop app. 
+If you are uncomfortable using Git, you can also check out <a href="https://youtu.be/RPagOAUx2SQ" title="YouTube: How to use GitHub Desktop: The easy tutorial(Part1)">this YouTube video</a> to do this all using the GitHub Desktop app.
```
Comment (sonnynomnom) [inline README.md:40]: LGTM

**Review by sonnynomnom — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sonnynomnom) [reply README.md:40]: Merging!

File: README.md (hunk 2 above)
Comment (JoshuaKGoldberg) [reply README.md]: TIL!

**Pull request merged**